### ISSUE TITLE
Adding redstone to any potion gives Corroded (1:00) potion

### ISSUE THREAD

**Issue by joshua-1403** [xJon/Hexxit-II#211]
**Basic details**
- **Are you using Java 8, 64-bit?**: yes
- **Did you add any additional mods**: no
- **Crash/latest log link (if applicable)**: not applicable

**Describe the issue**

After I downladed version 1.5.0, the vanilla brewing recipe to extend the night vision potion duration gives a corroded potion of the Fish's Undead Rising mod instead of a normal night vision with an 8 minute duration.
I tried it in a new creative world but got the same result.
Had Enough Items shows the expected result:
![2023-09-21_21 49 25](https://github.com/xJon/Hexxit-II/assets/145713832/3758a270-4c9b-48e4-b5fb-112e803f815a)
 
Brewing:
![2023-09-21_22 00 28](https://github.com/xJon/Hexxit-II/assets/145713832/5e99825a-1463-4cce-9a91-fc3d5fae79f5)

Result:
![2023-09-21_22 00 31](https://github.com/xJon/Hexxit-II/assets/145713832/311c9ecb-707b-4742-9a17-f558a87d7661)

I couldn't find something in a config file. The Fish's mod also has no entry that says it overwrites something.

Thanks in advance

**Comment by xJon** [comment]
Thank you for the report! Should be fixed in the next update thanks to the help of @IcarussOne.

**Comment by joshua-1403** [comment]
@xJon, the same happens with glowstone. There I get a Corroded II potion

**Comment by IcarussOne** [comment]
@joshua-1403
The recent v1.4.1 update to Fish's Undead Rising should fix this, until Hexxit II gets updated again I highly recommend updating from v1.4.0 in the meantime for a better experience. :)

**Comment by joshua-1403** [comment]
Nah, I'm fine. I can wait for the update. To find a solution I tested other stuff, there are a lot of mods I don't know what they do :D
The goggles from the Construct's Armory mod with night vision are good, currently I'm using night vision potions from the Reliquary mod. No need to hurry

**Comment by xJon** [comment]
This is resolved in update `1.6.0`!
